core/arch/asm_defines.asm

```diff
@@ -910,27 +910,71 @@ ASSUME fs:_DATA @N@\
         mov      ARG1, p1   @N@\
         blx      callee
 #elif defined(RISCV64)
+/* For RISC-V, there is no a instruction can operate on both immediates
```

Comment:
Looks like my prior requested replacements were not followed carefully enough.  Delete "a"; s/can/which can/.